packages/indicators/src/__tests__/__integration/integration.test.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { DataBroker } from '@tupaia/data-broker';
+import { runIntegrationTests } from '@tupaia/database';
+import { IndicatorApi } from '../../IndicatorApi';
+import { Analytic, FetchOptions } from '../../types';
+import arithmeticAnalyticsFixtures from './arithmeticAnalytics.fixtures.json';
+import indicatorApiFixtures from './indicatorApi.fixtures.json';
+
+interface TestCase {
+  input: {
+    indicatorCodes: string[];
+    fetchOptions: FetchOptions;
+  };
+  expected: Analytic[] | string;
+  throws?: boolean;
+}
+
+const dataBroker = new DataBroker();
+// TODO Use the base `@tupaia/database` models instead of `dataBroker.models`
+// after https://github.com/beyondessential/tupaia-backlog/issues/663 is implemented
```

Comment:
If this is because the `findOrDefault` behaviour is needed, I believe you pulled that into the central model recently?

Reply:
Correct! And I'm about to remove `findOrDefault` completely after implementing https://github.com/beyondessential/tupaia-backlog/issues/663, so I 'll update this after 663 ends in `dev`, which should be soon enough